src/redisearch_rs/inverted_index/src/lib.rs

```diff
@@ -273,6 +273,20 @@ impl<E: Encoder> InvertedIndex<E> {
         }
     }
 
+    /// Create a new inverted index from the given blocks and encoder. The blocks are expected to not
+    /// contain duplicate entries and be ordered by document ID.
```

Comment:
You can easily assert that, see https://github.com/RediSearch/RediSearch/blob/master/src/redisearch_rs/rqe_iterators/src/id_list.rs#L33